src/android/Capture.java

```diff
@@ -234,6 +246,17 @@ private void captureAudio(Request req) {
           try {
               Intent intent = new Intent(android.provider.MediaStore.Audio.Media.RECORD_SOUND_ACTION);
 
+              String timeStamp = new SimpleDateFormat("yyyyMMddHHmmssSSS").format(new Date());
+              String fileName = "AUDIO_" + timeStamp + ".wav";
```

Comment:
```suggestion
              String fileName = "cdv_media_capture_audio_" + timeStamp + ".m4a";
```

When I use the recording software, I get m4a files. I dont suggest changing format.